Doc/howto/urllib2.rst

```diff
@@ -594,5 +594,5 @@ This document was reviewed and revised by John Lee.
        scripts with a localhost server, I have to prevent urllib from using
        the proxy.
 .. [#] urllib opener for SSL proxy (CONNECT method): `ASPN Cookbook Recipe
-       <https://code.activestate.com/recipes/456195/>`_.
+       <https://code.activestate.com/recipes/456195-urrlib2-opener-for-ssl-proxy-connect-method/>`_.
```

Comment:
We should keep the diff as minimal as possible. Therefore, there is no need for a new line. You can keep it on one line.
No worries about the length of the line, our doc has a lines that are more longer than this :)
(This applies to the rest of the PR)

Reply:
Oh, good to know :smile: